packages/engine/src/avatar/functions/avatarFunctions.ts

```diff
@@ -86,6 +86,10 @@ export const boneMatchAvatarModel = (entity: Entity) => (model: Object3D) => {
 
   if (assetType == AssetType.FBX) {
     rootBone.children[0].scale.setScalar(0.01)
+    const object3DComponent = getComponent(entity, Object3DComponent)
+    if (object3DComponent.value) {
+      object3DComponent.value.userData.scale = 0.01
```

Comment:
we can avoid the conditional here by doing `object3DComponent.value!.userData.scale = 0.01` since we know Object3DComponent.value will be defined